### DISCUSSION TITLE
Pre-configured bond0 interface not showing up when doing network install-Cent7

### DISCUSSION THREAD

**Discussion by gransomware** [Security-Onion-Solutions/securityonion#3540] [General] [answered]
We're running into an issue when setting up the management server, which is pre-configured with two interfaces(em1/em2) bonded to the bond0 interface. The error that we're receiving is thrown by compare_main_nic_ip() function from so-functions. The error that is thrown: 
"The IP being routed by Linux is not the IP address assigned to the management interface ($MNIC).
This is not a supported configuration, please remediate and rerun setup."
This is failing because [[ "$MAINIP" != "$MNIC_IP" ]]

When I added $MAINIP to be thrown into /root/sosetup.log, it shows that it is failing the condition because it is picking up the correct IP from our bond0 interface, but when we're prompted to give the $MNIC from whiptail_management_nic() function from so-whiptail, it only shows our slave interfaces, em1/em2. I'm assuming that nic_list array from filter_unused_nics() isn't returning any bond0 interfaces that have been pre-configured prior to installation and isn't taking into account that they're slave interfaces, maybe?

Has anyone else ran into this issue or found any work arounds?

**Comment by tcritch05** [comment]
I am also having issues with this.

**Comment by palichx** [comment, marked as answer]
Hi,
looks like https://github.com/Security-Onion-Solutions/securityonion/issues/1289

in so-functions interface "Bond0" will excluded from array.
![image](https://user-images.githubusercontent.com/80513304/111966523-113ad680-8b08-11eb-8367-dae9b3182421.png)

_Has anyone else ran into this issue or found any work arounds?_

try to rename interface to bond3(for example)

**Reply by tcritch05** [reply]
Renaming the bonded interface to bond3 worked!
![image](https://user-images.githubusercontent.com/34383711/112152797-aabdc300-8bb0-11eb-9a78-6a06a5add024.png)
